.github/workflows/ci.yml

```diff
@@ -109,14 +109,21 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: make test
 
+      # We've ran `cargo build --all-targets` earlier, that is different to from just building the binary.
```

Comment:
But wouldn't `cargo build --all-targets` also build the binary? 🤔

Reply:
More details in the commit message :)
https://github.com/comit-network/comit-rs/pull/3215/commits/03adfb6a43d165b72b628fe647d97958a4c29e14